static/app/components/onboarding/gettingStartedDoc/onboardingCodeSnippet.tsx

```diff
@@ -32,11 +31,7 @@ export function OnboardingCodeSnippet(props: OnboardingCodeSnippetProps) {
   const [authTokenNodes, setAuthTokenNodes] = useState<HTMLSpanElement[]>([]);
 
   const handleAfterHighlight = useCallback((element: HTMLElement) => {
-    // Don't execute this code in tests as it will throw an error
-    // code snippet calls the onAfterHighlight callback too late, so it triggers the "updates should be wrapped into act(...)" error
-    if (NODE_ENV !== 'test') {
```

Comment:
It looks like this was already a problem here. I prefer mocking Prism to skipping in the test environment because it can be easily undone with jest.unmock() where necessary